app/views/planning_application/consultations/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :page_title do %>
+  Send letters to neighbours - <%= t('page_title') %>
+<% end %>
+
+<% add_parent_breadcrumb_link "Home", root_path(filter_options: PlanningApplication::FILTER_OPTIONS) %>
+<% add_parent_breadcrumb_link "Application", planning_application_path(@planning_application) %>
+<% add_parent_breadcrumb_link "Consultation", planning_application_path(@planning_application) %>
+<% content_for :title, "Send letters to neighbours" %>
+
+<%= render(
+  partial: "shared/proposal_header",
+  locals: { heading: "Send letters to neighbours" }
+) %>
+
+<%= render "shared/dates_and_assignment_header" %>
```

Comment:
Can you please delete this so it follows the same pattern as other views?

When I was reviewing the content and design of BOPS with Raffaella few months ago we said that is best to have the contents of `<%= render "shared/dates_and_assignment_header" %>` only in the parent directory such as `validation, assessment and review`.

Since you're adding a new step `Consultation` should definitely be there but not in the child. I know that some designs are not updated with this, so we should update them